src/test/java/org/kuali/kfs/sys/batch/BatchDirectoryHelper.java

```diff
@@ -0,0 +1,60 @@
+package org.kuali.kfs.sys.batch;
+
+import org.apache.commons.io.FileUtils;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.config.property.ConfigurationService;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Helper class to build up a batch directory and then tear it down when it is no longer needed.  To create a directory, call setUp; to remove
+ * the directory, just call tearDown.
+ */
+public class BatchDirectoryHelper {
+    private static org.slf4j.Logger LOG = org.slf4j.LoggerFactory.getLogger(BatchDirectoryHelper.class);
+
+    private String module;
+    private String directoryName;
+    private boolean batchFileDirectoryCreated = false;
+
+    private volatile static ConfigurationService configurationService;
+
+    public BatchDirectoryHelper(String module, String directoryName) {
+        this.module = module;
+        this.directoryName = directoryName;
+    }
+
+    public void setUp() {
```

Comment:
do we want to call these methods setUp() and tearDown() which has special meaning to JUnit, even though this isn't a test class, or should they be renamed to something more descriptive: createBatchDirectory(), delete/removeBatchDirectory()?